services/api-server/src/simcore_service_api_server/models/api_resources.py

```diff
@@ -46,6 +46,7 @@ def parse_last_resource_id(resource_name: RelativeResourceName) -> str:
 
 
 def compose_resource_name(*collection_or_resource_ids) -> RelativeResourceName:
+    # NOTE: collection_or_resource_ids expected de
```

Comment:
looks incomplete

Reply:
yep, it is indeed a type. Probably I ran out of ink ;-) :-D